Makefile

```diff
@@ -1,12 +1,11 @@
 # export SVN_REPO_BASE=. if you want to use the local version instead of trunk
 # from the subversion repository.
 
-# use something like "VERSION=0.2 make" to override the VERSION on the command line
-VERSION ?= $(shell sed -n "s/^.*[\t ]*VERSION[\t ]*=[\t ]*[\"']\([^\"']*\)[\"'].*/\1/gp" pycam/__init__.py)
+VERSION ?= $(shell git describe --tags | sed 's/^v//')
```

Comment:
This will pick the most recent tag, not just release tags (which I've been assuming begin with "v" followed by a number).  Also doesn't indicate in the version number if the working tree is dirty.  We're on the same track here, but I prefer this implementation: dcf1ffadff3a.